components/leaflet-maps/NormalMap.vue

```diff
@@ -17,7 +17,7 @@
 
             <div class="font-semibold text-base">{{ witch.name }}</div><br>
             <div>
-              <b>Investigation Date:</b> {{ witch.investigationDates[1] }}<br>
+              <b>Investigation Date:</b> {{ getFormatDate(witch.investigationDates[1],witch.precision) }}<br>
```

Comment:
Is there any risk here that we won't have a 1 index for the investigation dates? It would break the code if this is the case.

Reply:
There currently is no investigation dates with no index for investigation dates as they all have dates, but there is a risk that someone could remove one from wikidata. In this case the first index or investigation date would be saved as "N/A" in the APIDataHandler so I will account for that risk.

Reply:
Brilliant, thanks for confirming